pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -21,6 +21,13 @@ let
   buildKernel = any (n: n == configFile) [ "kernel" "all" ];
   buildUser = any (n: n == configFile) [ "user" "all" ];
 
+  # remove this patch at the next ZFS release
```

Comment:
That would be which version number?

Reply:
Hmm the next version? I'm not sure is there a release schedule plan? The patch/commit is already in upstream, so I guess the next minor/patch release like 2.0.2? I could write something like `... next ZFS release (> 2.0.1)`

Reply:
Yeah. Just say anything bigger than 2.0.1.

Reply:
Alright, I've pushed these changes